webui/react/src/pages/ExperimentDetails/ExperimentTrials.tsx

```diff
@@ -300,7 +300,7 @@ const ExperimentTrials: React.FC<Props> = ({ experiment }: Props) => {
           rowSelection={{
             onChange: handleTableRowSelect,
             preserveSelectedRowKeys: true,
-            selectedRowKeys: settings.row,
+            selectedRowKeys: settings.row ?? [],
```

Comment:
although `selectedRowKeys` is supposed to take in an `Array` or `undefined`, when `undefined` is passed in with the newer `antd 4.13+` version, it crashes antd `Table`.